source/agora/test/Base.d

```diff
@@ -902,6 +902,7 @@ public class TestNetworkManager : NetworkManager
         Params:
             conf = ban manager config
             clock = clock instance
+            _data_dir = path to the data directory
```

Comment:
Let's also rename the parameter to `data_dir` to be consistent with the `TestBanManager` constructor.